docs/reference/experiment-config-reference.rst

```diff
@@ -1483,38 +1483,19 @@ explicitly specified, the master will automatically generate an experiment seed.
  Profiling
 ***********
 
-The ``profiling`` section specifies configuration options related to profiling experiments. See
-:ref:`how-to-profiling` for a more detailed walkthrough.
+The ``profiling`` section specifies configuration options for the Determined system metrics
+profiler. See :ref:`how-to-profiling` for a more detailed walkthrough.
 
 ``profiling``
 =============
 
-Optional. Profiling is supported for all frameworks, though timings are only collected for
-``PyTorchTrial``. Profiles are collected for a maximum of 5 minutes, regardless of the settings
-below.
+Optional. Defaults to false.
```

Comment:
What does it do?

Reply:
Not being purposefully dense, I can't differentiate between:
```yaml
profiling:
  profiler: <val>
  enabled: <val>
```

Reply:
removed this section